### DISCUSSION TITLE
请问如何设置实现【快捷键唤出查词面板，并且查词面板处于活跃状态】？

### DISCUSSION THREAD

**Discussion by lasdocas** [crimx/ext-saladict#1658] [Q&A]
以便于唤出查词面板之后，可以直接输入单词，而不需要用鼠标点击一下查词面板。

提前感谢。

**Comment by crimx** [comment]
你的具体使用流程是如何的？默认查词面板出来应该是可以直接输入单词的。

**Reply by lasdocas** [reply]
当浏览器最大化的时候，使用我设置的快捷键（Alt+Z）唤出沙拉查词，浏览器的窗口和沙拉查词的窗口会自动并列，而这时候浏览器的窗口处于活跃状态，而沙拉查词窗口处于非活跃状态。

我刚发现，当浏览器没有最大化的时候（处于桌面任意位置），使用我设置的快捷键（Alt+Z）唤出沙拉查词，浏览器的窗口的大小也会变化，但是沙拉查词窗口处于活跃状态，可以直接输入。

不知道是不是快捷键冲突了……

我使用的是 Microsoft Edge 版本 98.0.1108.62 (官方内部版本) (64 位)。

**Reply by crimx** [reply]
感谢反馈，我调查一下
